### ISSUE TITLE
 Still waiting for db to be ready....

### ISSUE THREAD

**Issue by Fireobirdy** [commsy/commsy#130]
When i try to start commsy v10.0.4 i get the following error message:
`commsy-php-1    | Waiting for db to be ready...
commsy-php-1    | Still waiting for db to be ready... Or maybe the db is not reachable. 59 attempts left
`

I start commsy with the following command:
`sudo ./../.docker/cli-plugins/docker-compose -f docker-compose.yml up`

The last message from the db-container is:`commsy-db-1     | Version: '10.3.32-MariaDB-1:10.3.32+maria~focal'  socket: '/va                                                                                                        r/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
`

Any tipps?

**Comment by EofChris** [comment]
To start the application a `docker-compose up` from the root directory should be enough. When using elasticsearch you also need to append the related configuration: `docker-compose up -f docker-compose.yml -f docker/docker-compose.elastic.yml up`. Also note, that those are specific for development. When using in production you can grap the pre-releases container images.

Create a local file `.env.local` and set the right environment variable for the database connection. For a development environment this will be `DATABASE_URL=mysql://commsy:commsy@db:3306/commsy`. Check .env for reference.

**Comment by Fireobirdy** [comment]
Thank you for the tipp to edit the .env-file. After that, php-container was able to connect to the database but complained about missing tables in the database. So I copied the file initial.sql to the db-container and imported its content to the db. When I now browse the webserver on port 81, I get the following error message:
`An exception has been thrown during the rendering of a template ("Could not find the entrypoints file from Webpack: the file "/var/www/html/public/build/entrypoints.json" does not exist.").`

I spent 20h+ to get commsy running and even tested version 8 and 9 without success. I learnd some new stuff about docker, but at this point, I have to give up. My colloquium about learning and teaching in digital spaces, where I wanted to introduce commsy, is on saturday and I dont have any time left. I don't know if you guys have imagined, that someone is keen enough to try setting up commsy without a documentation. Anyway, this issue can be closed.
